map_merge/RobotPose[] poses
sensor_msgs/PointCloud2[] maps
